examples/loitering/demo.ipynb

```diff
@@ -0,0 +1,356 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filter_cons = {\n",
+    "    \"__class__\": lambda x: x == Person,\n",
+    "    \"bottom_center\": vqpy.utils.continuing(\n",
+    "        condition=vqpy.utils.within_regions([REGION]), duration=10, name=\"in_roi\"\n",
+    "    ),\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Selecting VObjs' properties for output"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For each person, we can output its tracking id, coordinate, and duration of staying in the region:\n",
+    "\n",
+    "- tracking id can be selected with `track_id`\n",
+    "- outputting coordinate needs to select `coordinate` then serialize the ndarray to string with post-processing function `lambda x: str(x)`\n",
+    "- `in_roi_periods` can be used to access list of time periods (in seconds) of VObjs satisfying the condition\n",
+    "\n",
+    "\tthis name comes from appending `\"_periods\"` to `name` defined in `vqpy.utils.continuing`"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The select condition of the query could be:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "select_cons = {\n",
+    "    \"track_id\": None,\n",
+    "    \"coordinate\": lambda x: str(x),  # convert to string for JSON serialization\n",
+    "    # name in vqpy.continuing + '_periods' can be used in select_cons.\n",
+    "    \"in_roi_periods\": None,\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The query"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inheriting `vqpy.QueryBase`, we can compose a query using the `filter_cons, select_cons` created earlier:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class People_loitering_query(vqpy.QueryBase):\n",
+    "    @staticmethod\n",
+    "    def setting() -> vqpy.VObjConstraint:\n",
+    "        return vqpy.VObjConstraint(\n",
+    "            filter_cons, select_cons, filename=\"loitering\"\n",
+    "        )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running the query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "video_path = \"./loitering.mp4\"\n",
+    "save_folder = \"./vqpy_outputs\"\n",
+    "detector_model_dir = \"./\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Next, we can run the query by `vqpy.launch`. 
For `cls_name`, ...
For `cls_type`, ...